doc/releases/release-notes-3.6.rst

```diff
@@ -322,6 +365,8 @@ Drivers and Sensors
 
   * Renesas R-Car pinctrl driver now supports Gen4 SoCs
   * Renamed ``CONFIG_PINCTRL_RA`` to :kconfig:option:`CONFIG_PINCTRL_RENESAS_RA`
+  * On STM32 devices, pins assigned to JTAG/SW port can now be put to analog state when
+    :kconfig:option:`CONFIG_PM=y` and :kconfig:option:`CONFIG_DEBUG=n`
```

Comment:
You can't have the `=y` and `=n` parts in the references.

Reply:
TIL